File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int h;
    do
    {
        h = get_int("Altura: ");
    }
    while(h < 1 || h > 8);
    
    for(int i = 1; i <= h; i++)
    {
        for(int d = h - i; d > 0; d--)
        {
            printf(" ");
        }
        for(int hash = h - (h - i); hash > 0; hash--)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario/less/mario.py
from cs50 import get_int

height = 0
c = 1

while 1 > height or height > 8:
    height = get_int('Height: ')

while height > 0:
    print(' ' * (height - 1), end = '')
    print('#' * c)
    height -= 1
    c += 1